packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -516,6 +518,13 @@ class CupertinoTextField extends StatefulWidget {
   /// Has no effect when [suffix] is null.
   final OverlayVisibilityMode suffixMode;
 
+  /// Controls the vertical aligment of the [suffix] and the [prefix] widget in relation to content.
+  ///
+  /// Defaults to [CrossAxisAlignment.center].
+  ///
+  /// Has no effect when the [suffix] and the [prefix] is null.
```

Comment:
"is null" => "are null"

Reply:
What i should be do here? 
What is null and what are null? prop => suffix or suffix => prop?